src/darker/tests/test_command_line.py

```diff
@@ -256,20 +256,6 @@ def test_parse_command_line(
     dict(config={"check": True}, expect_warn=set()),
     dict(config={"isort": True}, expect_warn=set()),
     dict(config={"lint": ["pylint"]}, expect_warn=set()),
-    dict(
-        config={"skip_string_normalization": True},
-        expect_warn={
-            "The configuration option `skip_string_normalization` in [tool.darker] is"
-            " deprecated and will be removed in Darker 3.0."
-        },
-    ),
-    dict(
-        config={"skip_magic_trailing_comma": True},
-        expect_warn={
-            "The configuration option `skip_magic_trailing_comma` in [tool.darker] is"
-            " deprecated and will be removed in Darker 3.0."
```

Comment:
I would love for the information that this is moved to `[tool.black]` now to be included here, so I don't have to research that when getting the warning.